lib/hyrax/configuration.rb

```diff
@@ -373,6 +373,12 @@ def subject_prefix
       @subject_prefix ||= "Contact form:"
     end
 
+    attr_writer :extract_full_text
+    def extract_full_text?
+      return @extract_full_text unless @extract_full_text.nil?
+      @extract_full_text = true
+    end
```

Comment:
I think this would be a reasonable change, but the above follows the pattern used throughout this particular source. I would suggest a specific change/issue to fix this seperately IMHO.